assembl/templates/views/navBarRight.tmpl

```diff
@@ -20,6 +21,7 @@
     </li>
     {% endif %}
     {% endif %}
+    {% endif %}
```

Comment:
Good point, and certainly worth a comment somewhere, but has nothing to do with my PR, as it's not new code. Fixing everything within 10 lines of any change fulfills the definition of scope creep. (Same with  code that was moved in a PR, btw.) (Sorry, vaguely a rant, but you're right we need a process for such comments. PR may not be it.)